shared/deps/run_autotest/boottool.py

```diff
@@ -199,6 +199,9 @@ def get_packed(self):
         params.append(0)
         params.append(self.attributes)
 
+        if isinstance(params[0], six.string_types):
+            params[0] = params[0].encode()
```

Comment:
All the string elements of `params` should be encoded.

https://docs.python.org/3.6/library/struct.html?highlight=struct%20pack#struct.pack